Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CopySample.ps1

```diff
@@ -0,0 +1,196 @@
+﻿#===========================================================================
+# UWP Community Toolkit Sample Copier
+# -----------------------------------
+# Run script using PowerShell to copy a sample app folder with a new name
+# to use as a template for a new sample.
+#
+# Will automatically update name of classes to new name.
+#
+# Tool will pop-up a window which lets you select an existing sample.
+# Enter a new sample name and a category, then click Copy.
+#===========================================================================
+
+# Reference: https://foxdeploy.com/2015/04/16/part-ii-deploying-powershell-guis-in-minutes-using-visual-studio/
+
+# Load Required Reference
+[void][System.Reflection.Assembly]::LoadWithPartialName('presentationframework')
+
+# Get Current Sample List
+$samples = Get-ChildItem -Directory | Get-ItemPropertyValue -Name Name
```

Comment:
If I run this through ps or cmd, I usually run it from the root of the repo. Could this be hardcoded to always grab the samples in the SamplePages directory regardless where it's invoked from? You can then also keep this in the root of the sample app so it's easier to run.

Reply:
Right now it assumes it's run from the location it's in.  Which if you right-click and run from File Explorer should be the case.

I think I can grab the 'script location' and then use that as a starting point instead.  I'll take a look.  I can move it to be next to the readme in the root.  I'll make an update.

Reply:
@nmetulev fixed now, the script is smart now and will run when executed from any other path (though it still needs to be located in the sample app root).

Added a blurb to the readme too.